components/automate-dex/web/templates/login.html

```diff
@@ -10,25 +10,23 @@
     <link href="{{ url .ReqPath "static/main.css" }}" rel="stylesheet">
     <link href="{{ url .ReqPath "theme/styles.css" }}" rel="stylesheet">
   </head>
-  <div class="theme-panel">
-    <div class="logo">
-      <img class="theme-navbar__logo" src="{{url .ReqPath "static/img/automate-blue-d9789f4b.svg" }}">
-    </div>
-    <div>
-      {{ range $c := .Connectors }}
-      <div class="theme-form-row">
-        <a href="{{ $c.URL }}" target="_self">
-          <div class="dex-btn theme-btn-provider">
-              <span class="dex-btn-icon dex-btn-icon--{{ $c.ID }}"></span>
-              {{ if eq $c.ID "local" }}
-              <span class="dex-btn-text">Sign in as a local user</span>
-              {{ else }}
-              <span class="dex-btn-text">Sign in with {{ $c.Name }}</span>
-              {{ end }}
-          </div>
-        </a>
+  <body>
+    <main class="theme-panel">
+      <div class="logo">
+        <img class="theme-navbar__logo" alt="Chef Company Logo" src="{{url .ReqPath "static/img/automate-blue-d9789f4b.svg" }}">
```

Comment:
suggest: `alt="Chef - Home"`

Reply:
can you say more about why? ill want to update this for automate-ui as well

Reply:
ooh huge mistake here - thought that was a `link` - hadn't fired up the site yet.

I'd suggest just `alt="Chef"`

Reply:
As far as the Why - I think its important to note, and you probably already know this, but for anyone else reading.. that what you have now is not wrong.

In my opinion the toughest part of a11y is that there is a lot of ambiguity in the best practices of it.  I tend to lean in favor of removing any description that isn't already known.  In this case, the Logo is sitting in an <img> tag, so we already know its an image.  And we know we are on the Chef software site.  

You could also argue that Chef is too common and would be misleading in that it might be a Chef of the cooking variety.

I guess all of this is to say - thats just my opinion on the fact - But I'm open to being convinced either way 🤷‍♂️ 

https://inviqa.com/blog/writing-effective-alt-text-images

Reply:
found this too: https://webaim.org/techniques/alttext/#logos

Reply:
exactly 😂  - i like https://www.deque.com/accessible-development/ a lot as well.

Reply:
oh wait this should prolly be Chef Automate now that i think about it to make it distinct from the other products.. will update again